include/c2pa.h

```diff
@@ -408,10 +436,10 @@ int c2pa_builder_add_resource(struct C2paBuilder *builder_ptr,
  * Reads from NULL-terminated C strings.
  */
 IMPORT extern
-int c2pa_builder_add_ingredient(struct C2paBuilder *builder_ptr,
-                                const char *ingredient_json,
-                                const char *format,
-                                struct CStream *source);
+int c2pa_builder_add_ingredient_from_stream(struct C2paBuilder *builder_ptr,
```

Comment:
Would this be an API breaking change because of the renaming?

Reply:
I added a comment in the PR about this.